seqr/views/utils/dataset_utils.py

```diff
@@ -97,7 +97,7 @@ def load_mapping_file_content(file_content):
 def match_sample_ids_to_sample_records(
         projects,
         user,
-        sample_ids,
+        sample_id_to_tissue_type,
```

Comment:
Pass both sample ids and the mapping from sample ids to tissue types to the function.